### PR TITLE
Fixes #19697 - Change method for job template imports

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: app/models/job_template_importer.rb [hunk 1]
```diff
@@ -5,16 +5,34 @@
...
+    if template && template.locked? && !template.new_record? && !force
+      return { :old => template.template_was,
+               :new => template.template,
+               :status => false,
+               :result => "Skipping Template #{to_id_string template}:#{name} - template is locked" }
+    end
+    nil
+  end
+
+  def self.to_id_string(template)
```
Comment (iNecas) [inline app/models/job_template_importer.rb]: Minor: `id_string` would read a bit better than `to_id_string`

File: app/models/job_template_importer.rb [hunk 2]
```diff
@@ -5,16 +5,34 @@
...
+  def self.skip_locked(name, force)
+    template = JobTemplate.find_by :name => name
+
+    if template && template.locked? && !template.new_record? && !force
+      return { :old => template.template_was,
+               :new => template.template,
+               :status => false,
+               :result => "Skipping Template #{to_id_string template}:#{name} - template is locked" }
+    end
+    nil
```
Comment (iNecas) [inline app/models/job_template_importer.rb]: The `return` and `nil ` is not necessary

**Comment by iNecas** [comment]
Looks reasonable, few comments + failing tests

**Comment by xprazak2** [comment]
I fixed the rubocop issues and addressed the comments.

**Comment by adamruzicka** [comment]
Test failures seem unrelated, let's get this in

**Comment by iNecas** [comment]
Thanks @xprazak2

**Pull request merged**